src/status_im/ethereum/subscriptions.cljs

```diff
@@ -82,4 +82,7 @@
     "wallet-owned-collectibles-filtering-done" {:fx [[:dispatch
                                                       [:wallet/owned-collectibles-filtering-done
                                                        event]]]}
+    "wallet-get-collectibles-details-done"     {:fx [[:dispatch
```

Comment:
will it be possible to move this file to status-im2 soon?

Reply:
Not sure, but anyway I'd rather make it a separate refactoring PR